### PR TITLE
Created Header content

### PR DESCRIPTION
**Pull request by shahnawaza75**
Created html css fils.
Created linters.
Adeed images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# Myfirst_Porfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+**Portfolio**
+
```
Comment (ShoiraTa) [inline README.md:4]: - [ ] Please consider updating your readme file adding more information and properly aligning sections. Currently readme looks very confusing:

Your README  | An example of a professional README
------------- | -------------
Screenshot 2022-03-17 at 20 45 43  | Screenshot 2022-03-17 at 20 45 31

[Here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) you can read more and learn about formatting `.md` documents on Github

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+**Authors**
+:silhouette:  **Shahnawaz Ali**
+- GitHub: [@githubhandle](https://github.com/shahnawaza75)
+- Twitter: [@twitterhandle](https://twitter.com/RjShahnawaza75)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shahnawaz-ali-a24b72204)
+ **:handshake: Contributing**
+Contributions, issues, and feature requests are welcome!
+Give a :star:️ if you like this project!
+ **:memo: License**
+This project is [MIT](./MIT.md) licensed.
```
Comment (ShoiraTa) [inline README.md:21]: - [ ] This link is currently broken, please consider fixing it by adding MIT license file into the project

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @shahnawaza75 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
Hi @shahnawaza75, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**